gui/elements/qt/_layer_buttons.py

```diff
@@ -1,29 +1,31 @@
-from PyQt5.QtCore import Qt, QSize
+from PyQt5.QtCore import Qt
 from PyQt5.QtGui import QIcon
 from PyQt5.QtWidgets import QWidget, QVBoxLayout, QHBoxLayout, QPushButton, QFrame, QCheckBox
-from ._layerList import QtLayerList
-from os.path import dirname, join, realpath
-from numpy import empty
+from os.path import join
+from ...icons import icons_dir
 
-dir_path = dirname(realpath(__file__))
-path_delete = join(dir_path,'icons','delete.png')
-path_add = join(dir_path,'icons','add.png')
-path_off = join(dir_path,'icons','annotation_off.png')
-path_on = join(dir_path,'icons','annotation_on.png')
+path_delete = join(icons_dir,'delete.png')
+path_add = join(icons_dir,'add.png')
```

Comment:
At this rate, it’d be easier to just create a new func that adds to the `icons` path.

Reply:
Sounds good, maybe you want to give that a go in a future PR?